troposphere/static/js/actions/QuotaActions.js

```diff
@@ -0,0 +1,45 @@
+define(function (require) {
+
+  var Utils = require('./Utils');
+      Router = require('../Router'),
+      Constants = require('constants/QuotaRequestConstants'),
+      QuotaRequest = require('models/QuotaRequest');
+
+  return {
+      //create: function (params) {
```

Comment:
Is there a use case to keep the create and destroy actions, or can we remove the commented code?